SECURITY.md

```diff
@@ -0,0 +1,42 @@
+# GRASS GIS Security Policy
+
+## Reporting a Vulnerability
+At GRASS GIS, we take security vulnerabilities seriously. We appreciate your efforts in responsibly disclosing any issues you may find. To report a security vulnerability, please follow these steps:
+1. **Privately disclose the issue** by emailing our [Security Team](vpetras@ncsu.edu). Please do not create publicly viewable issues for security vulnerabilities.
```

Comment:
Do we want to offer the GitHub way to privately report a security vulnerability too as an alternative? If it enabled here? It's hard for people with already write access to see it, and I don't have a second account to test it.

https://docs.github.com/en/code-security/security-advisories/guidance-on-reporting-and-writing-information-about-vulnerabilities/privately-reporting-a-security-vulnerability